### ISSUE TITLE
Add Giveth organization and add GnosisDAO & Bankless links

### ISSUE THREAD

**Issue by hantuzun** [neu-fi/turkey-relief-dao#30]
https://giveth.io/project/gnosisdao-earthquake-relief
https://giveth.io/project/banklessdao-turkey-disaster-relief-fund

**Comment by tervelix** [comment]
1- add 'giveth' option;

```
`const OPTION_TYPES = [
  'cryptocurrency',
  'bank',
  'btcturk',
  'card',
  'nft',
  'giveth',
];`
```

2- add giveth icon:

```
const icons: {[index: string]:any} = {
  cryptocurrencies: {
    'Bitcoin': 'https://raw.githubusercontent.com/spothq/cryptocurrency-icons/master/svg/color/btc.svg',
    'Ethereum': 'https://raw.githubusercontent.com/spothq/cryptocurrency-icons/master/svg/color/eth.svg',
    'Avalanche': 'https://raw.githubusercontent.com/spothq/cryptocurrency-icons/master/svg/color/avax.svg',
    'Binance': 'https://raw.githubusercontent.com/spothq/cryptocurrency-icons/master/svg/color/bnb.svg',
    'Polygon': 'https://raw.githubusercontent.com/spothq/cryptocurrency-icons/master/svg/color/matic.svg',
    'Solana': 'https://raw.githubusercontent.com/spothq/cryptocurrency-icons/master/svg/color/sol.svg',
    'Cosmos': 'https://raw.githubusercontent.com/spothq/cryptocurrency-icons/master/svg/color/atom.svg',
    'Polkadot': 'https://raw.githubusercontent.com/spothq/cryptocurrency-icons/master/svg/color/dot.svg',
    'Tron': 'https://raw.githubusercontent.com/spothq/cryptocurrency-icons/master/svg/color/trx.svg',
    'Mina': "https://cryptologos.cc/logos/mina-mina-logo.svg",
    'Cardano': "https://raw.githubusercontent.com/spothq/cryptocurrency-icons/master/svg/color/ada.svg",
    'VeChain': "https://raw.githubusercontent.com/spothq/cryptocurrency-icons/master/svg/color/vet.svg",
    'Waves': "https://raw.githubusercontent.com/spothq/cryptocurrency-icons/master/svg/color/waves.svg",
    'Tezos': "https://raw.githubusercontent.com/spothq/cryptocurrency-icons/master/svg/color/xtz.svg",
    'Celo': "https://cryptologos.cc/logos/celo-celo-logo.svg",
    'Gnosis': "https://cryptologos.cc/logos/gnosis-gno-gno-logo.svg",
    'Arbitrum': "./icons/arbitrum.svg"
    'Optimism': "https://cryptologos.cc/logos/optimism-ethereum-op-logo.svg"
  },
  bank: './icons/bank.svg',
  btcturk: './icons/btcturk.svg',
  card: './icons/card.svg',
  nft: './icons/nft.svg',
  giveth: './icons/giveth.svg
};
```

3 - add organizations

```
  {
    name: 'Gnosis DAO',
    description: 'Team of experts within Gnosis Chain and its ecosystem and currently contributing in the Community, Communication and Marketing team.',
    logoUrl: 'https://cryptologos.cc/logos/gnosis-gno-gno-logo.svg',
    websiteUrl: 'https://www.gnosis.io/gnosisdao',
    twitterUrl: 'https://twitter.com/GnosisDAO',
    popularity: 1,
    endorsementUrls: ['https://twitter.com/Givethio/status/1623493391395815428?s=20&t=hTSxUyujyIj_o4ucioeNNA'],
    categories: ['international', 'dao'],
    options: [
      {
        type: 'giveth',
        name: 'Giveth',
        linkName: 'Donating with cryptocurrencies via Giveth',
        linkUrl: 'https://giveth.io/donate/gnosisdao-earthquake-relief',
      },
      {
        type: 'card',
        name: 'Card',
        linkName: 'Option of Credit Card via Giveth',
        linkUrl: 'https://giveth.io/donate/gnosisdao-earthquake-relief',
      },
    ],
  },
  {
    name: 'Bankless DAO',
    description: 'BanklessDAO is a decentralized community with one mission: Help the world.',
    logoUrl: './icons/bankless.svg',
    websiteUrl: 'https://www.bankless.community/',
    twitterUrl: 'https://twitter.com/banklessDAO',
    popularity: 1,
    endorsementUrls: ['https://twitter.com/Givethio/status/1623493391395815428?s=20&t=hTSxUyujyIj_o4ucioeNNA'],
    categories: ['international', 'dao'],
    options: [
      {
        type: 'giveth',
        name: 'Giveth',
        linkName: 'Donating with cryptocurrencies via Giveth',
        linkUrl: 'https://giveth.io/donate/gnosisdao-earthquake-relief',
      },
      {
        type: 'card',
        name: 'Card',
        linkName: 'Option of Credit Card via Giveth',
        linkUrl: 'https://giveth.io/donate/gnosisdao-earthquake-relief',
      },
    ],
  },
```

**Comment by hantuzun** [comment]
Thanks @tervelix.

Could you update linkNames as short call to action prompts? They're links as you know.

Could you also find 3 round (strongly preferred SVG) icons for Giveth, Bankless, and Gnosis DAO?

To the React.js developers; building the filter functionality to handle Giveth as another "other" donation type is the missing and the tricky part.

**Comment by oytuncoban** [comment]
Closing as resolved with #88.
